DNN Platform/Tests/DotNetNuke.Tests.Urls/TestUtil.cs

```diff
@@ -86,7 +86,7 @@ internal static string FilePath
         internal static string GetEmbeddedFileName(string fileName)
         {
             string fullName = String.Format("{0}.{1}", EmbeddedFilePath, fileName);
-            if (!fullName.ToLower().EndsWith(".csv"))
+            if (!fullName.ToLowerInvariant().EndsWith(".csv"))
```

Comment:
Back to my initial question. Are you sure that we need to change test code to fix a bug about upgrading DNN?